server/config/toml.go

```diff
@@ -25,7 +25,7 @@ const DefaultConfigTemplate = `
 
 # Tracer defines the 'vm.Tracer' type that the EVM will use when the node is run in
 # debug mode. To enable tracing use the '--evm.tracer' flag when starting your node.
-# Valid types are: json|struct|access_list|markdown
+# Valid types are: json|struct|access_list|markdown|firehose
```

Comment:
Hi @Eduard-Voiculescu 
Could you add something in app.go that uses this config to set the required tracer?